src/utils/consts.go

```diff
@@ -0,0 +1,13 @@
+package utils
+
+import cappv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+
+var cappAPIGroup = cappv1alpha1.GroupVersion.Group
```

Comment:
This is already defined here https://github.com/dana-team/platform-backend/blob/42cd557e3785424ba6c893f8274ea22a092f2ad8/src/utils/labels.go#L9

Reply:
I moved it here